src/lib/pure_pursuit/PurePursuit.hpp

```diff
@@ -124,7 +139,14 @@ class PurePursuit : public ModuleParams
 		float lookahead_min{1.f};
 	} _params{};
 private:
-	float _lookahead_distance{0.f}; // Radius of the circle around the vehicle
-	Vector2f _distance_on_line_segment{}; // Projection of prev_wp_to_curr_pos onto prev_wp_to_curr_wp
-	Vector2f _curr_pos_to_path{}; // Shortest vector from the current position to the path
+	/**
+	 * @brief Publish pure pursuit message
+	 */
+	void publishPurePursuit();
+
+	float _target_bearing{NAN};
+	float _lookahead_distance{NAN}; // Radius of the circle around the vehicle
+	float _crosstrack_error{NAN}; // Shortest distance from the current position to the path (Positiv: right of path, Negativ: left of path)
+	Vector2f _curr_pos_to_curr_wp{};
+	Vector2f _distance_along_path{}; // Projection of prev_wp_to_curr_pos onto prev_wp_to_curr_wp
```

Comment:
_distance_along_path I would assume is a single float, denoting a "distance". Maybe reconsider _position_along_path?